code/game/gamemodes/cult/rune_spawn_action.dm

```diff
@@ -46,6 +46,8 @@
 	if(damage_interrupt && isliving(owner))
 		var/mob/living/L = owner
 		health = list("health" = L.health)
+	if(istype(get_turf(owner), /turf/open/floor/engine/cult))
+		scribe_time = scribe_time * 0.5
```

Comment:
it will, just needs copying into a local variable and using that variable in the ```do_after()```